modules/services/sql/rds.tf

```diff
@@ -1,9 +1,12 @@
-data "aws_vpc" "default" {
-  default = true
+data "aws_subnet_ids" "default" {
+  vpc_id = var.specific_vpc_id
+  count = var.specific_subnet_group_name == "" ? 1 : 0
```

Comment:
Hey @toxinu, during deployment of a fresh instance, I've stumbled with an issue here.

I didn't define a specific VPC ID because I wanted to go with a default one and got the following error:
```
│ Error: Missing required argument
│ 
│   with module.sql.aws_db_subnet_group.primary[0],
│   on .terraform/modules/sql/modules/services/sql/rds.tf line 8, in resource "aws_db_subnet_group" "primary":
│    8:   subnet_ids = tolist(data.aws_subnet_ids.default[0].ids)
│ 
│ The argument "subnet_ids" is required, but no definition was found.
```

Shouldn't `vpc_id` here be defined like:
```
locals {
  vpc_id = var.specific_vpc_id != "" ? var.specific_vpc_id : data.aws_vpc.default[0].id
}

data "aws_vpc" "default" {
  default = true
  count = length(var.specific_subnet_ids) > 0 ? 0 : 1
}

data "aws_subnet_ids" "default" {
  vpc_id = local.vpc_id
  count = length(var.specific_subnet_ids) > 0 ? 0 : 1
}
```
?